### PR TITLE
VIH-5255 Fix for Endpoint Assertion

### PR DESCRIPTION
**Pull request by npoxon**
### Change description ###
- Fix for Endpoint Assertion

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs [hunk 1]
```diff
@@ -40,10 +40,16 @@ private static void AssertEndpoints(IReadOnlyCollection<EndpointResponse2> heari
         {
             foreach (var conferenceEndpoint in conferenceEndpoints)
             {
-                var hearingEndpoint = hearingEndpoints.First(x => x.Id.Equals(conferenceEndpoint.Id));
+                var hearingEndpoint = hearingEndpoints.First(x => x.Sip.Equals(conferenceEndpoint.Sip_address));
+ 
+                if (conferenceEndpoint.Defence_advocate == null)
+                {
+                    hearingEndpoint.Defence_advocate_id.Should().BeNull();
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs:47]: would it not be best to assert the defence advocate id is that of a participant in the same conference?

**Review by shaed-parkar — COMMENTED** [review]
just a comment about the asseertion

**Review by RuwanikaWeerakkody — APPROVED** [review]

File: AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs (hunk 1 above)
Comment (npoxon) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs:47]: What is public string Defence_advocate { get; set; } on the hearing endpoint response? Is it display name?

Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs:47]: I believe so. may have to double-check the contract or check the mapper

Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs:47]: looks like it's the username

Comment (npoxon) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Data/AssertConference.cs:48]: @shaed-parkar Updated!

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**